src/ProjectDataBase/Script.PostDeployment1.sql

```diff
@@ -231,7 +231,7 @@ BEGIN TRY
 
     PRINT 'Inserting seed data for Status table'
 
-    INSERT INTO dbo.[Status]([StatusName])
+    INSERT INTO dbo.[TicketStatus]([StatusName])
     VALUES
         ('ok'),
```

Comment:
It is worth to have values like Ok, Error (PascalCase) because you will have such enum values